core/src/main/java/org/infinispan/configuration/cache/CacheMode.java

```diff
@@ -5,6 +5,11 @@
  */
 public enum CacheMode {
    /**
+    * Data is not replicated and certain features are not possible
```

Comment:
I think this would be a good place to document what features are possible...

OTOH, I'm not sure adding a new cache mode is really the best way to expose this.